packages/transaction-controller/package.json

```diff
@@ -55,10 +55,12 @@
     "@metamask/auto-changelog": "^3.4.4",
     "@metamask/ethjs-provider-http": "^0.2.0",
     "@types/jest": "^27.4.1",
+    "@types/jest-when": "^2.7.3",
```

Comment:
should this be on at least the next major to match the actual dependency?

https://www.npmjs.com/package/@types/jest-when

Reply:
Yes, probably. There are a couple of other packages which are also using this for tests, so I just reused the same version to satisfy the Yarn constraints. I'd have to update those too. What do you think if I did this in a separate PR?

Reply:
ahhh gotcha. a problem for later then 💯